matter_server/server/device_controller.py

```diff
@@ -1076,15 +1090,15 @@ async def _setup_node(self, node_id: int) -> None:
             ):
                 try:
                     await self.interview_node(node_id)
-                except (NodeNotResolving, NodeInterviewFailed) as err:
+                except (NodeNotResolving, NodeInterviewFailed, TimeoutError) as err:
                     LOGGER.warning("Unable to interview Node %s", exc_info=err)
                     # NOTE: the node will be picked up by mdns discovery automatically
                     # when it comes available again.
                     return
             # setup subscriptions for the node
             try:
                 await self._subscribe_node(node_id)
-            except NodeNotResolving:
+            except (NodeNotResolving, TimeoutError):
```

Comment:
...and here?

This way we know if this was aborted due to timeout or some other problem,

Reply:
added additional logging info